### ISSUE TITLE
bower dependency

### ISSUE THREAD

**Issue by Faryshta** [ekaragodin/yii2-bootstrap-slider#2]
hi, why you are not loading the bootstraps slider js and css from a bower dependency on the composer.json?

**Comment by ekaragodin** [comment]
Hi, when I did the package have not yet had the possibility set bower dependencies in package.json. I'll do it in the future or your can make pull request :)

**Comment by Faryshta** [comment]
yes i can. just one question. how is this plugin different from http://www.yiiframework.com/doc-2.0/yii-bootstrap-carousel.html ?
